docs/source/markdown/podman-create.1.md

```diff
@@ -974,11 +974,75 @@ Remote connections use local containers.conf for defaults
 Set the umask inside the container. Defaults to `0022`.
 Remote connections use local containers.conf for defaults
 
-#### **\-\-uidmap**=*container_uid:host_uid:amount*
+#### **\-\-uidmap**=*container_uid*:*from_uid*:*amount*
```

Comment:
You referred to the host_uid below, and I am not sure from_uid is helpful here.

Reply:
@rhatdan I'm not sure I follow. At least I'm not using the text _host_uid_.

```
[esjolund@laptop podman]$ grep host_uid ./docs/source/markdown/podman-create.1.md
[esjolund@laptop podman]$ grep host_uid ./docs/source/markdown/podman-run.1.md 
[esjolund@laptop podman]$ 
```

I'm using _from_uid_ more lika a variable name. The name could be something else too.

As I understand it, the value given at this position has two
quite different meanings. In the rootful case the value is a host UID. In the rootless case it's not a host UID. 
Just to give it a name I called it an "intermediate UID". Maybe this is more of an internal implementation detail, but I think it will be easier for a newbie user to understand how to use **--uidmap** by giving it a name. It will also be easier to write the documentation.

Reply:
One way to avoid using _from_uid_ could be to write something like:

#### **\-\-uidmap**=map

Valid _map_ values are

*  *container_uid*:*host_uid*:*amount* 
*   *container_uid*:*intermediate_uid*:*amount*

The first version is used when podman is called by root. The second version is used when podman is called by a non-root user.

Reply:
I think if you put something like:

```
The from_uid value is based upon the user running the command, either rootful or rootless users.  
  * rootful user:    container_uid:host_uid:amount
  * rootless user: container_uid:intermediate_uid:amount
```